src/components/ImportForm/ReviewSection/RuntimeSelector.tsx

```diff
@@ -45,22 +46,43 @@ export const RuntimeSelector: React.FC<RuntimeSelectorProps> = ({ detectedCompon
   const [selectedRuntime, setSelectedRuntime] = React.useState(null);
 
   const DetectingRuntime = 'Detecting runtime...';
+  const dockerFileSample = React.useMemo(
+    () => ({
+      key: 'docker-build',
+      value: 'Dockerfile',
+      icon: <DockerIcon color="#2496ed" />,
+      runtime: { name: 'Dockerfile' },
+    }),
+    [],
+  );
+
   const items = React.useMemo(() => {
     return (
-      samples?.map((s) => ({ key: s.uid, value: s.name, icon: <img src={s.icon.url} /> })) || []
+      [
+        ...(samples || []).map((s) => ({
+          key: s.uid,
+          value: s.name,
+          icon: <img src={s.icon.url} />,
+        })),
+        dockerFileSample,
+      ] || []
```

Comment:
It was discussed that if a dockerfile is detected, we shouldn't remove the dockerfile (previously named 'Other') option. But only if the dockerfile was detected should we provide this option.

This will allow the user to _always_ choose dockerfile as an option. But that's not a viable solution right now without the ability for the user to set the dockerfile location and build context. I believe that without those inputs, there is no way that this option would work if a dockerfile wasn't already detected.

Maybe delivering this right now is ok because we plan to support these new inputs soon.